src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -282,10 +290,10 @@ def acr_task_update(cmd,  # pylint: disable=too-many-locals
     if task.trigger:
         source_triggers = task.trigger.source_triggers
         base_image_trigger = task.trigger.base_image_trigger
-        if commit_trigger_enabled or source_triggers is not None:
-            status = None
-            if commit_trigger_enabled is not None:
-                status = TriggerStatus.enabled.value if commit_trigger_enabled else TriggerStatus.disabled.value
+        if (commit_trigger_enabled or pull_request_trigger_enabled) or source_triggers is not None:
```

Comment:
This is not specifically in this PR, do you want to check if `source_triggers is not None` or if source_triggers is not empty? /cc @ankurkhemani

Reply:
After offline discussion, we are changing this from null check to empty check. See most recent update.

Reply:
RP returns null but sure we can check for empty to be sure.